sdks/go/pkg/beam/model/fnexecution_v1/beam_fn_api.pb.go

```diff
@@ -27,7 +27,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.27.1
-// 	protoc        v4.24.4
+// 	protoc        v3.21.12
```

Comment:
is this protoc downgrade intentional or do you need to update your protoc version?

Reply:
This is the output of go generate. Is there another documented way to generate the proto?

Reply:
It's the correct way to generate but you likely have an older version of protoc installed. IIRC the script doesn't download and install+ build that.

Reply:
Thank you @randymaldonado and @lostluck for catching this <3 Updated with latest protoc.

Reply:
I suggest matching the `protoc` version to have it be consistent with the rest of the codebase.

Running `go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.31.0` should update it. v1.31.0 should be used because this matches what is in the go.mod for sdks ([link](https://github.com/apache/beam/blob/master/sdks/go.mod#L65C29-L65C36)).